recipe/vs2010_omp.patch

```diff
@@ -0,0 +1,12 @@
+--- Source/LibRawLite/libraw/libraw_types.h
++++ Source/LibRawLite/libraw/libraw_types.h
+@@ -32,7 +32,7 @@
+ #if defined (_OPENMP)
+ 
+ #if defined(WIN32)
+-# if defined (_MSC_VER) && (_MSC_VER >= 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
++# if defined (_MSC_VER) && (_MSC_VER > 1600 || (_MSC_VER == 1500 && _MSC_FULL_VER >= 150030729) )
```

Comment:
@almarklein do you know anything about this patch?  I don't recall putting this in.  It seems you were the original author at conda-recipes: https://github.com/conda/conda-recipes/blob/master/freeimage/vs2010_omp.patch